.github/workflows/static_checks.yml

```diff
@@ -16,6 +16,9 @@ jobs:
         with:
           fetch-depth: 2
 
+      - name: Install Node.js
```

Comment:
Pretty sure this part can be removed now that everything is handled via pre-commit

Reply:
You are right. We don't need to install a specific version of node and from what I've checked, in the `ubuntu-22.04` image there is a `node 18.20.2 LTS` available by default with `npm 10.5.0` which will do the job.

Reply:
AFAIK the node version on the existing image shouldn't matter at all.

The only dependency for our pre-commit hooks is Python + `pre-commit`. Everything else gets downloaded by pre-commit and configured in the appropriate versions to match the configuration. So it's not using the image's node, it's using a custom one.